client/src/components/HomePage.js

```diff
@@ -12,14 +13,7 @@ const HomePage = (props) => {
                 </div>
             </div>
             <div className="cell small-12 medium-12 large-6">
-                <div className="container">
-                    <h1>Popular destinations</h1>
-                    <ul>
-                        <li>list of popular cities in united states (links to a show page)
-                            render another component here that fetches the list from another source
-                        </li>
```

Comment:
I see that this was removed/ replaced - in the future I do not recommend "placeholder" or "filler" text. Better to leave it blank until you build the feature